sql/table.cc

```diff
@@ -3927,7 +3927,7 @@ static uint find_field(Field **fields, uchar *record, uint start, uint length)
 
 	/* Check that the integer is in the internal */
 
-int set_zone(register int nr, int min_zone, int max_zone)//TIANMU UPGRADE 
+int set_zone(int nr, int min_zone, int max_zone)//TIANMU UPGRADE 
```

Comment:
why remove register?

Reply:
The c++17 standard deprecates the register keyword